source/documentation/3.9/rspec-core/file.Changelog.html

```diff
@@ -56,7 +57,23 @@
         <div class="clear"></div>
       </div>
 
-      <div id="content"><div id='filecontents'><h3>3.9.1 / 2019-12-28</h3>
+      <div id="content"><div id='filecontents'><h3>Development</h3>
+
+<p><a href="http://github.com/rspec/rspec-core/compare/v3.9.1...master">Full Changelog</a></p>
+
+<p>Bug Fixes:</p>
+
+<ul>
+<li>Emit a warning when <code>around</code> hook is used with <code>:context</code> scope
+(Phil Pirozhkov, #2687)</li>
+<li>Prevent invalid implementations of <code>Exception#cause</code> from being treated as a
+valid cause (and causing strange errors) in <code>RSpec::Core::Formatters::ExceptionPresenter</code>.
+(Jon Rowe, #2703)</li>
+<li>Correctly detect patterns when <code>rspec_opts</code> is an array in <code>RSpec::Core::RakeTask</code>.
+(Marc-André Lafortune, #2704)</li>
+</ul>
+
+<h3>3.9.1 / 2019-12-28</h3>
```

Comment:
You've generated these from the wrong commit / branch, I think that explains the other changes...

Reply:
Yes. [The task](https://github.com/rspec/rspec-dev/blob/master/Rakefile#L66) get latest commit `"git checkout #{args[:branch]} && git pull --rebase` 
https://github.com/rspec/rspec-dev/commit/e03b43e977ed8c35b991a58c0f879b6656ff7660

Maybe the command is only run when we release a new version and the maintenance branch does not commit that are not in a release?

What about automaticly selecting last release using `git` `tags`?
```diff
--- a/Rakefile
+++ b/Rakefile
@@ -63,7 +63,7 @@ desc "Updates the rspec.github.io docs"
 task :update_docs, [:version, :branch, :website_path] do |t, args|
   abort "You must have ag installed to generate docs" if `which ag` == ""
   args.with_defaults(:website_path => "../rspec.github.io")
-  run_command "git checkout #{args[:branch]} && git pull --rebase"
+  run_command "git fetch --tags && git checkout `git tag | grep #{args[:version]} | tail -1`"
```

For this result:
```
=== rspec-mocks =============================
.....

HEAD is now f45810aa Version 3.9.1

=== rspec-support ===========================
.....

HEAD is now 17a63b5 Version 3.9.2

==================================================
```

Reply:
The patch for this proposal https://github.com/rspec/rspec-dev/pull/247

The result seems much better https://github.com/rspec/rspec.github.io/compare/no-plugin-from-source...no-plugin-from-source-with-last-release?expand=1#diff-93e4a181d3b66d096e3ed3077744ba7bR60

Reply:
Sounds good, give it a go!